addons/spreadsheet/static/src/global_filters/components/filter_text_value/filter_text_value.xml

```diff
@@ -2,7 +2,9 @@
 <templates>
     <t t-name="spreadsheet.TextFilterValue">
         <div class="o_input o-global-filter-text-value d-flex flex-wrap gap-1">
-            <TagsList tags="tags"/>
+            <t t-foreach="this.tags" t-as="tag" t-key="tag.id">
+                <BadgeTag text="tag.text" onDelete="tag.onDelete"/>
+            </t>
```

Comment:
Out of curiosity, what's the reason behind this change ? Why not just keep `TagsList` ?